front/lib/models/assistant/agent.ts

```diff
@@ -113,10 +113,6 @@ export class AgentConfiguration extends Model<
 
   declare maxToolsUsePerRun: number;
 
-  declare generationConfigurationId: ForeignKey<
```

Comment:
The column is already removed on prod from this commit: https://github.com/dust-tt/dust/pull/4712/files#diff-766f5c1abd936709ce451f717525d138ff51a578da653997be29a431d9441b81L204-L225

but this was not removed

Reply:
AFAIU deleting this line is what actually deletes the column in DB. Done in #4802